Code/GraphMol/FileParsers/atropisomers_catch.cpp

```diff
@@ -0,0 +1,123 @@
+//
+//  Copyright (C) 2025 Greg Landrum and other RDKit contributors
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include <string>
+
+#include "RDGeneral/test.h"
+#include <catch2/catch_all.hpp>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/Chirality.h>
+#include <GraphMol/FileParsers/FileParsers.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/SmilesParse/SmilesWrite.h>
+#include <GraphMol/test_fixtures.h>
+
+using namespace RDKit;
+
+TEST_CASE("explicit tests") {
+  // making some of the other atropisomer tests explicitly test what we expect
+  std::string rdbase = getenv("RDBASE");
+  REQUIRE(rdbase != "");
+  SECTION("basics") {
+    std::string file =
+        rdbase +
+        "/Code/GraphMol/FileParsers/test_data/atropisomers/AtropManyChirals.sdf";
+    std::unique_ptr<RWMol> mol(MolFileToMol(file));
+    REQUIRE(mol);
+    CHECK(mol->getBondWithIdx(7)->getStereo() ==
+          Bond::BondStereo::STEREOATROPCCW);
+  }
+  SECTION("macrocycle") {
+    std::string file =
+        rdbase +
+        "/Code/GraphMol/FileParsers/test_data/atropisomers/macrocycle-5-meta-Cl-ortho-hash.sdf";
+    std::unique_ptr<RWMol> mol(MolFileToMol(file));
+    REQUIRE(mol);
+    CHECK(mol->getBondWithIdx(15)->getStereo() ==
+          Bond::BondStereo::STEREOATROPCW);
+  }
+  SECTION("with stereo groups and other chiral centers") {
+    REQUIRE(rdbase != "");
+    std::string file =
+        rdbase +
+        "/Code/GraphMol/FileParsers/test_data/atropisomers/AtropManyChiralsEnhanced.sdf";
+    std::unique_ptr<RWMol> mol(MolFileToMol(file));
+    REQUIRE(mol);
+    CHECK(mol->getBondWithIdx(7)->getStereo() ==
+          Bond::BondStereo::STEREOATROPCCW);
+    CHECK(mol->getAtomWithIdx(6)->getChiralTag() ==
+          Atom::ChiralType::CHI_UNSPECIFIED);
+    CHECK(mol->getAtomWithIdx(9)->getChiralTag() !=
+          Atom::ChiralType::CHI_UNSPECIFIED);
+    CHECK(mol->getAtomWithIdx(13)->getChiralTag() !=
+          Atom::ChiralType::CHI_UNSPECIFIED);
+    CHECK(mol->getAtomWithIdx(16)->getChiralTag() !=
+          Atom::ChiralType::CHI_UNSPECIFIED);
+    CHECK(mol->getAtomWithIdx(10)->getChiralTag() !=
+          Atom::ChiralType::CHI_UNSPECIFIED);
+
+    CHECK(mol->getStereoGroups().size() == 2);
+    for (const auto &sg : mol->getStereoGroups()) {
```

Comment:
The first stereo group for this mol should have a bond included for the atropisomer.  This code works:
    auto sg = mol->getStereoGroups()[0];
    CHECK(sg.getAtoms().size() == 2);
    CHECK(sg.getBonds().size() == 1);
    sg = mol->getStereoGroups()[1];
    CHECK(sg.getAtoms().size() == 2);
    CHECK(sg.getBonds().size() == 0);